frontend/pages/app/offline-page.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+	<div class="offline-container">
+	  <h1>You are offline</h1>
+	  <p>Please check your internet connection.</p>
+	  <button @click="tryReconnect">Try Reconnecting</button>
+	</div>
+  </template>
+  
+  <script>
+  export default {
+	data() {
+	  return {
+		// Store a reference to the bound method so it can be added and removed consistently
+		boundRedirectUser: null,
+	  };
+	},
+	mounted() {
+	  this.boundRedirectUser = this.redirectUser.bind(this);
```

Comment:
I would use an arrow function here instead of .bind
`this.boundRedirectUser = () => this.redirectUser()`

I wonder if there's a better way to unset a listener. Does addEventListener have a return value?

Reply:
As far as I have seen "addEventListener" doesn't have a return value